src/Organisms/RegistrationForm/RegistrationForm.js

```diff
@@ -0,0 +1,111 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import PriSecBtn from '../../Atoms/PriSecBtn/PriSecBtn';
+import CardWrapper from '../../Atoms/CardWrapper/CardWrapper';
+import RegistrationFormStyled from './RegistrationFormStyled';
+import InputField from '../../Molecules/InputField/InputField';
+import CheckboxAgreeField from '../../Atoms/CheckboxAgreeField/CheckboxAgreeField';
+
+const RegistrationPage = () => {
```

Comment:
2 important things to think about:
UNO:  validation of inputs (what kind of data yuo expect from the user, is it number, string, how long, what format, maybe regex could help)
DOS: error handling, what if the form was not filled correctly, what if the registration did not go through, becasue the server responded with an error and user could not be registered. What does the user see in that case?

Reply:
Thanks for all reviews. Should i prepare client-side validation ? Maybe i can do it in time with server side validation ?